paddle/fluid/framework/dlpack_tensor_test.cc

```diff
@@ -91,23 +91,11 @@ void TestMainLoop() {
     }
   }
 }
+TEST(dlpack, test_all) {
```

Comment:
It seems that we cannot use `test_##cpp_type` in the nested macro. Just move all tests into one function.